views.go

```diff
@@ -63,7 +63,7 @@ func LoginGET(c echo.Context) error {
 
 	if username.String() == "" && verification_key.String() == "" {
 		data.LoginForm = true
-		data.CASLogin = GetCasLoginUrl(service.String())
+		data.CASLogin = GetCasLoginUrl(c.Request(), service.String())
```

Comment:
Ok so I'm not sure I fully understand: the PR doesn't modify the way fakecas is creating URLs (relative or otherwise).  It's maintaining existing behavior.

Do you want me to change `c.Request()` to a `string` containing the host name?